src/ApiService/ApiService/TestHooks/NotificationOperationsTestHooks.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System.Net;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.OneFuzz.Service;
+using Microsoft.OneFuzz.Service.OneFuzzLib.Orm;
+
+
+#if DEBUG
+namespace ApiService.TestHooks {
+    public class NotificationOperationsTestHooks {
+
+        private readonly ILogTracer _log;
+        private readonly IConfigOperations _configOps;
+        private readonly INotificationOperations _notificationOps;
+
+        public NotificationOperationsTestHooks(ILogTracer log, IConfigOperations configOps, INotificationOperations notificationOps) {
+            _log = log.WithTag("TestHooks", nameof(NotificationOperationsTestHooks));
+            _configOps = configOps; ;
+            _notificationOps = notificationOps;
+        }
+
+        [Function("NewFilesTestHook")]
+        public async Task<HttpResponseData> NewFiles([HttpTrigger(AuthorizationLevel.Anonymous, "put", Route = "testhooks/notificationOperations/newFiles")] HttpRequestData req) {
```

Comment:
This might be too meta but do you think it's worthwhile to use a source generator to auto create these?

Reply:
you are right, i was thinking of dealing with that later. 

I was thinking of doing Azure API management and then work from there at some point. 

At this point I'm willing to type things